packages/cli/templates/default-plugin/package.json.hbs

```diff
@@ -1,11 +1,12 @@
 {
-  "name": "@backstage/plugin-{{id}}",
+  "name": "{{name}}",
   "version": "{{version}}",
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "Apache-2.0",
-  "private": true,
+  "private": {{privatePackage}},
```

Comment:
These additions are breaking the `plugin:diff` command, as it execute the templates and then compare output. We need to add logic to the `diff` part of the cli to read out the config values for each plugin, and then supply them to the templating. That stuff is happening here: https://github.com/spotify/backstage/blob/87699ffe202eff366a8932f7b1926c7c965905f9/packages/cli/src/commands/plugin/diff.ts#L78

Reply:
i'm working on it.

Reply:
We also can't extract `version` from lerna.json. Plugins will eventually have a different version from backstage.

https://github.com/spotify/backstage/blob/f90d645461bf82f5745c262e042671c6b04cb362/packages/cli/src/commands/plugin/diff.ts#L65